### PR TITLE
[TTAHUB-3483] checkbox to activity reports

### PR DESCRIPTION
**Pull request by thewatermethod**
## Description of change
Add checkbox to switch between types of goals on activity reports

## How to test
- Change is deployed to sandbox
- Confirm that you can switch to "OHS standard goals" on activity report. Confirm that if a grant has existing responses for the FEI goal, they appear in the goal form. Confirm that CRUD operations for both types of goals work as normal.

## Issue(s)
* https://jira.acf.gov/browse/TTAHUB-3483

### Every PR

- [ ] Meets issue criteria
- [ ] JIRA ticket status updated
- [ ] Code is meaningfully tested
- [ ] Meets accessibility standards (WCAG 2.1 Levels A, AA)
- [ ] API Documentation updated
- [ ] Boundary diagram updated
- [ ] Logical Data Model updated
- [ ] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions
- [ ] UI review complete

### Before merge to main

- [ ] OHS demo complete
- [ ] Ready to create production PR

### Production Deploy

- [ ] Staging smoke test completed

### After merge/deploy

- [ ] Update JIRA ticket status

### CODE REVIEW

File: src/goalServices/goals.js [hunk 1]
```diff
@@ -683,7 +683,6 @@ export async function goalsForGrants(grantIds) {
       'endDate',
       'source',
       'createdVia',
-      [sequelize.fn('BOOL_OR', sequelize.literal(`"goalTemplate"."creationMethod" = '${CREATION_METHOD.CURATED}'`)), 'isCurated'],
```
Comment (AdamAdHocTeam) [inline src/goalServices/goals.js:686]: this is no longer needed?

**Review by AdamAdHocTeam — COMMENTED** [review]
So if I select a regular goal then check the 'use standard goal box' my original selection is not cleared on the UI. Users might find this confusing.

**Comment by thewatermethod** [comment]
> So if I select a regular goal then check the 'use standard goal box' my original selection is not cleared on the UI. Users might find this confusing (and vice versa).

Do you mean clearing the goal or just the value in the select box?

**Review by AdamAdHocTeam — APPROVED** [review]
Code looks good. And the question of check box behavior seems to have ben answered in stand up.

**Pull request merged**